pkg/operator/configobserver/network/observe_network.go

```diff
@@ -0,0 +1,111 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/ghodss/yaml"
+	"github.com/golang/glog"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"k8s.io/client-go/listers/core/v1"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
```

Comment:
@sttts @deads2k we hardcode this in so many places, I almost wish we have some const package in library-go to control this from one place?

Reply:
> @sttts @deads2k we hardcode this in so many places, I almost wish we have some const package in library-go to control this from one place?

no const package